demo/meeting/README.MD

```diff
@@ -20,9 +20,31 @@ This is the SDK react meeting demo app. It shows how to use the Amazon Chime SDK
 
 ### Troubleshooting
 
+- You can change the `amazon-chime-sdk-js` [log levels](https://aws.github.io/amazon-chime-sdk-js/enums/loglevel.html) by providing `logLevel` as a query parameter in the URL.
+  - WARN logs (default)
```

Comment:
To avoid making the readme too long, can we reduce this to something like 

`You can adjust the log levels by including a "logLevel" param with one of the following values: 'warn', 'info', etc...

See the MeetingProvider docs [link to docs] for more information around logging"

Reply:
Is this fine? 
- You can change the `amazon-chime-sdk-js` [log levels](https://aws.github.io/amazon-chime-sdk-js/enums/loglevel.html) by providing `logLevel` as a query parameter with one of these values 'warn' | 'error' | 'info' | 'debug' | 'off'.
   ```
    https://0.0.0.0:9000/?logLevel=debug
    ```